public/app/plugins/datasource/cloudwatch/datasource.ts

```diff
@@ -562,9 +563,11 @@ export default class CloudWatchDatasource extends DataSourceApi<CloudWatchQuery,
         return { ...result, [key]: value };
       }
 
-      const valueVar = this.templateSrv.variables.find(({ name }) => name === this.templateSrv.getVariableName(value));
+      const valueVar = this.templateSrv
+        .getVariables()
+        .find(({ name }) => name === this.templateSrv.getVariableName(value));
       if (valueVar) {
-        if (valueVar.multi) {
+        if (((valueVar as unknown) as VariableWithMultiSupport).multi) {
```

Comment:
Why not just `if (variable && (variable as VariableWithMultiSupport).multi)`?

Reply:
I think it will be a strict null error, if I'm not misstaken.

Reply:
Yes, the `as unknown` is a strict null error bypass.